src/validate.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-lines */
+function validate(targetWord, toCase) {
+  const SUPPORTED_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errorsArray = [];
+  const TEXT_TO_CONVERT_ERROR =
```

Comment:
All this constants need to be in one place. Move it to separate file, then import. 
Or move outside function without status code ( if you not create separate file for all constants) 
const STATUS_CODE_400 
const STATUS_CODE_200
const TEXT_TO_CONVERT_ERROR
const SUPPORTED_CASES